src/DataViews/SamplingReportDataViewTest.cpp

```diff
@@ -138,12 +144,16 @@ std::string GetExpectedDisplayAddressByIndex(size_t index) {
   return absl::StrFormat("%#llx", kSampledAbsoluteAddresses[index]);
 }
 
-std::string GetExpectedDisplayFunctionNameByIndex(size_t index, CaptureData& capture_data) {
-  return capture_data.GetFunctionNameByAddress(kSampledAbsoluteAddresses[index]);
+std::string GetExpectedDisplayFunctionNameByIndex(size_t index, const ModuleManager* module_manager,
+                                                  CaptureData& capture_data) {
+  return orbit_client_data::GetFunctionNameByAddress(*module_manager, capture_data,
+                                                     kSampledAbsoluteAddresses[index]);
 }
 
-std::string GetExpectedDisplayModuleNameByIndex(size_t index, CaptureData& capture_data) {
-  std::string module_path = capture_data.GetModulePathByAddress(kSampledAbsoluteAddresses[index]);
+std::string GetExpectedDisplayModuleNameByIndex(size_t index, const ModuleManager* module_manager,
+                                                CaptureData& capture_data) {
```

Comment:
Pass a const reference? Also can `capture_data` be a const ref as well (I know it was already like that)?

Reply:
Sure, good point. Missed that in the previous pass. Also made capture_data const ref.